src/app/demo/demo.component.html

```diff
@@ -1,6 +1,33 @@
 <div class="demo-page">
   <div class="nav-side-menu">
-    <div class="brand">angular-archwizard</div>
+    <div class="brand">
+      <a class="navbar-brand text-white" href="#">
+        <img src="https://angular.io/assets/images/logos/angular/angular.svg" width="32px" height="32px"/>
+        angular-archwizard
+      </a>
+
+      <div class="btn-group btn-group-toggle d-block mx-auto dir" data-toggle="buttons">
+        <label
+          class="btn btn-secondary"
+          (click)="changeDirTo('ltr')">
+          <input type="radio" name="options" id="ltr" checked> LTR
+        </label>
+        <label
+          class="btn btn-secondary"
+          (click)="changeDirTo('rtl')">
+          <input type="radio" name="options" id="rtl"> RTL
+        </label>
+      </div>
+
+      <form class="form mx-auto github">
+        <a class="github-button"
+           href="https://github.com/madoar/angular-archwizard"
+           data-icon="mark-github"
+           data-size="large"
+           data-show-count="true"
+           aria-label="Visit madoar/angular-archwizard on GitHub">Github</a>
+      </form>
```

Comment:
Should this be part of the header or the footer of the navigation bar?
I think it looks a bit out of place where it is now.

Reply:
I moved the github button to the current footer of the page with few changes.
I think you will like it :).